pcweb/components/navbar.py

```diff
@@ -182,40 +181,43 @@ class NavbarState(State):
     page_score: int = 0
 
     show_form = False
+    form_submitted = False
 
     current_category = "All"
 
     def handle_submit(self, form_data: dict):
-        self.feedback = form_data["feedback"]
+        feedback = form_data["feedback"]
 
         # Check if the email is valid.
         if "email" in form_data:
             self.email = form_data["email"]
-            try:
-                validation = validate_email(self.email, check_deliverability=True)
-                self.email = validation.email
-            except EmailNotValidError as e:
-                # Alert the error message.
-                return rx.window_alert(str(e))
+
+        if len(feedback) < 10:
+            return rx.window_alert("Please enter your feedback. (min 10 characters)")
 
         current_page_route = self.get_current_page()
-        # Check if the user is already on the waitlist.
-        with rx.session() as session:
-            # Add the feedback to database.
-            session.add(
-                Feedback(
-                    email=self.email,
-                    feedback=self.feedback,
-                    score=self.score,
-                    page=current_page_route,
-                ),
-            )
-            session.commit()
-            print("session")
-            # contact_data = json.dumps({"email": self.email})
-            # self.add_contact_to_loops(contact_data)
+
+        discord_message = f"""
+_________________________
+Contact: {self.email}
+Feedback: {self.feedback}
+Score: {"?" if  self.page_score==0 else "👍" if self.page_score > 1 else "👎"}
+Page: {current_page_route}
+_________________________
+"""
+
+        DISCORD_WEBHOOK_URL = os.environ.get("DISCORD_WEBHOOK_URL")
+        payload = {"content": discord_message}
+        try:
+            response = requests.post(DISCORD_WEBHOOK_URL, json=payload)
+        except:
+            pass
+
+        self.show_form = False
+        self.form_submitted = True
 
     def update_score(self, score):
+        print("update score", score)
```

Comment:
stray print

Reply:
Yes need todo a few more updates not completely ready yet I'll mark it in draft